backend/seeds/003_reviews.js

```diff
@@ -0,0 +1,34 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex("reviews")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("reviews").insert([
+        {
+          rating: 2,
+          comments: "Mediocre chicken.",
+          user_id: 1,
+          location_id: 1
+        },
+        {
+          rating: 3,
+          comments: "Best chicken I've ever had!",
+          user_id: 1,
```

Comment:
Obviously not a huge deal for release one but we generally won't want two reviews of the same location by the same user (I assume). Something we'll want to consider when we get to the add review stage.

Reply:
I thought about this, and I don't think there's anything wrong with the person not liking the stuff the first time around and then coming back later on and potentially liking it then. That would be the only case I can think of where 2 reviews by the same user of the same location would be used.

Ultimately, you're right though. We can figure that out later on if we want to put that limitation there.

Reply:
What about just updating the original review? Or at least some kind of limit on how many reviews one user can submit.